example/lib/bloc/map_bloc.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:bloc/bloc.dart';
+import 'package:rxdart/rxdart.dart';
+import 'bloc.dart';
+
+class MapBloc extends Bloc<MapEvent, MapState> {
+  @override
+  MapState get initialState => MapState.initial();
+
+  @override
+  Stream<MapState> transformEvents(
+      Stream<MapEvent> events, Stream<MapState> Function(MapEvent event) next) {
+    return (events as Observable<MapEvent>)
```

Comment:
This should be updated to filter and only debounce `MapCameraPositionChanged` events. 
More info here: https://github.com/felangel/bloc/issues/524

Reply:
Thank you! applied it!